### PR TITLE
automated release process

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: delivery.yaml [hunk 1]
```diff
@@ -72,10 +71,34 @@ pipeline:
...
     - ~/.cache/go-build # Go build cache
   commands:
-  - desc: push
+  - desc: create tag and release
     cmd: |
-      VERSION=$(git describe --tags --always)
+      LATEST_VERSION=$(git describe --tags --always)
+      VERSION_PART=$(cat VERSION)
+      OLD_PATCH=$(echo $LATEST_VERSION | awk -F . '{print $3}')
+      NEW_PATCH=$((OLD_PATCH + 1))
```
Comment (aermakov-zalando) [inline delivery.yaml]: This needs to reset somehow when we increment `VERSION`.

File: delivery.yaml [hunk 2]
```diff
@@ -72,10 +71,34 @@ pipeline:
...
+      VERSION=${VERSION_PART}${NEW_PATCH}
```
[the remark this answers is not in the document]
Comment (szuecs) [reply delivery.yaml]: ah shit you are right, thanks

File: delivery.yaml (hunk 1 above)
Comment (szuecs) [reply delivery.yaml]: true

**Comment by aermakov-zalando** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**